tutorials/marble/getting-started.md

```diff
@@ -0,0 +1,56 @@
+-[The Marble Platform](#marble-platform)
+-[Nodes](#nodes)
+- [Main Areas of the Platform](#main-areas)
+  - [The Homepage](#homepage)
+  - [JupyterLab](#jupyterlab)
+  - [The Data Catalog](#data-catalog)
+  - [The Web Processing Services](#webservices)
+
+
+## <a id="marble-platform"></a>The Marble Platform
+
+Marble provides the perfect podium for climate researchers to store and share their work. 
+A climate researcher's findings can be stored on the Marble Platform where it will be accessible by the public.
+
+The Marble Platform is made up of several servers (nodes) connected to each other. 
+Each node provides data storage, and web services for managing workflows and processing data.  Logging into one node 
+allows access to the resources of all the nodes on the network.   
+
+## <a id="nodes"></a>Nodes
+
+A node is a server with specialized software installed and is hosted by a partner in the Marble network.
+There are currently four nodes in the Marble network, and each node has a different focus - the Ouranos PAVICS node 
```

Comment:
Please don't write this sort of thing in the tutorials, Every time we add a new node this information will need to be updated. Also the PCIC node isn't even in the network yet so this number is currently inaccurate.

Please remove this whole paragraph. If you need an introductory paragraph please talk about nodes and the network in the abstract without giving specific numbers or descriptions that are still very fluid.

Please change this to something like:

Reply:
Removed